src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-CornFlowerBlue: #1c64f2;
+$theme-Biscay: #1c325e;
+$theme-white: #ffffff;
+$theme-FrenchRose: #f1416c;
+$theme-FunBlue: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-PaleSky: #6b7280;
+$theme-Mischka: #d1d5db;
+$theme-AthensGray: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.875rem;
```

Comment:
why do we have so many decimal places in the margin?

Reply:
fixed